InventorySales/Model/AreProductsSalableForRequestedQty/ProductsSalableResult.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Model\AreProductsSalableForRequestedQty;
+
+use Magento\Framework\Api\AttributeValueFactory;
+use Magento\Framework\Api\ExtensionAttributesFactory;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Model\AbstractExtensibleModel;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Model\ResourceModel\AbstractResource;
+use Magento\Framework\Registry;
+use Magento\InventorySalesApi\Api\Data\ProductsSalableResultExtensionInterface;
+use Magento\InventorySalesApi\Api\Data\ProductsSalableResultInterface;
+
+/**
+ * @inheritDoc
+ */
+class ProductsSalableResult extends AbstractExtensibleModel implements ProductsSalableResultInterface
```

Comment:
Please, remove the inheritance of `\Magento\Framework\Model\AbstractExtensibleModel` and just implement extension attributes getter and setter. You may find a good example here `\Magento\InventoryInStorePickup\Model\PickupLocation`.
By this change, we will resolve MHI failure.

Reply:
done.